src/utilities/MiscUtils.ts

```diff
@@ -1,19 +1,105 @@
-export function stringToBoolean(str: string) {
-  switch (str.toLowerCase().trim()) {
-    case 'true' || 'yes' || '1' || 'on':
+type className = string | undefined | null;
```

Comment:
@charlieporth1 can you think of a more clear name for this utils class? maybe we need to break the methods out into separate classes.

Reply:
@spawn-dev can do